examples/demos/random-specimen.bot

```diff
@@ -0,0 +1,13 @@
+for x,y in grid(1, 6, 10, 60):
+  randomfont = choice(fontnames())
+
+  # font title
+  fill(0.5)
+  font("Liberation Sans", 14)
+  text(randomfont, 15, y+30)
+
+  # specimen line
+  fill(0.2)
+  font(randomfont, 36)
+  text("Handgloves", 25, y+65)
+
```

Comment:
This indentation doesn't look like PEP8 format (8 spaces), you can run `black` to format a bot by pointing it at the bot directly: `black examples/demos/random-specimen.bot`

Reply:
I didn't even notice, Atom just let me go like that and Shoebot didn't complain either :)